website/src/pages/_app.tsx

```diff
@@ -46,5 +59,18 @@ function MyApp({ Component, pageProps: { session, cookies, ...pageProps } }: App
     </>
   );
 }
-export { getServerSideProps };
+
+MyApp.getInitialProps = ({ ctx: { req } }: AppContext) => {
+  return {
+    env: {
+      INFERENCE_SERVER_HOST: process.env.INFERENCE_SERVER_HOST,
+      ENABLE_CHAT: isChatEnabled(),
+      ENABLE_EMAIL_SIGNIN: boolean(process.env.ENABLE_EMAIL_SIGNIN),
+      ENABLE_EMAIL_SIGNIN_CAPTCHA: boolean(process.env.ENABLE_EMAIL_SIGNIN_CAPTCHA),
+      CLOUDFLARE_CAPTCHA_SITE_KEY: boolean(process.env.CLOUDFLARE_CAPTCHA_SITE_KEY),
+    },
+    cookie: req?.headers.cookie,
```

Comment:
can you check if this is still needed? I remember that I did some debugging, and this cookie field was always empty.

Reply:
The req object only available when we use getServerSideProps. So it still works if the page is used with getServerSideProps.

Reply:
So now we are using mixed getStaticProps and getServerSideProps, it only partially works. I think that's fine